.github/workflows/r.yml

```diff
@@ -112,7 +112,7 @@ jobs:
       fail-fast: false
       matrix:
         config:
-          - { org: "rstudio", image: "r-base", tag: "4.0-centos7" }
+          - { org: "rstudio", image: "r-base", tag: "4.1-centos7" }
           - { org: "rhub", image: "debian-gcc-devel", tag: "latest" }
```

Comment:
I know this changes the way this is setup right now, but I wonder if it would make sense to have this build with centos + debian be R old release (4.0) and devel respectively and then have the ubuntu job up above be our current release job? 

That would at least segregate the release from non-release, and make sure we're testing release on a modern(ish) OS compared to centos7 which has some interesting caveats due to its build system (it's good that we test centos7 here and not only in nightly, but it feels a little weird that that's our one linux release test).

Reply:
It does sound like it makes more sense to be testing on an OS that is more reflective of where more users are likely to be.